posts/lost.md

```diff
@@ -1,9 +1,9 @@
 # Lost in Thought
 
-Why is called *lost in thought*?
+Why is called _lost in thought_?
```

Comment:
Why is it called _lost in thought_?

Reply:
Will have to bring this up with copywriters at the next standup. 

cc: Head of Copy @michaelnyu thoughts?

Reply:
looping in CCO (Chief Copywriter Officer) @wongbryan.  hope to get a decisions made before EOW thanks -

Reply:
I'm for this change, will have to confirm with CFCP (Casca Fan Club President) @epicchewy. Thoughts on approving this?

Reply:
Thank you for bringing this to my attention. This seems to be a large step in the right direction. It has my approval!

Reply:
Got it, making update. Good work boys.